### ISSUE TITLE
Raft becomes unactionable

### ISSUE THREAD

**Issue by robert-bowman** [anegostudios/VintageStory-Issues#2423]
### Game Version

v1.18r3

### Platform

Windows

### Modded

None

### SP/MP

Singleplayer

### Description

raft was fine for several trips until I left it in some reeds and moved about harvesting them.  Move at most 30 block away.  When I returned the overlay and crosshair both recognized the raft, but I could not get on it.  Could not damage it.  block beneath it, registered the hits.

### How to reproduce

I will try

### Logs

```Log```

**Comment by robert-bowman** [comment]
Update:  Went back and found my raft.  Used it for a couple of more trips and it again became unusable.  I did notice a behavior.  If I have an oar in right hand.  The raft moves away from me when I near it.  If I don't have the oar in hand I cannot interact with it.

**Comment by robert-bowman** [comment]
Update:  I moved several hundred blocks away from the raft and as I returned, I was within 50 blocks of the raft when I was teleported onto the raft.  It was functional and I used it to return to my base.

**Comment by Th3Dilli** [comment]
thank you for your report

did this happen on a server or hosted LAN session?

if so then this should be the exact same issue as the linked on

**Comment by QuantumToasted** [comment]
Chiming in - I've noticed that the raft seems to become inactionable when I have something in my offhand or a full inventory and try to pick it up. This happens on a local/singleplayer server, have not tested in multiplayer. It seems to eventually unstick itself after 30 seconds or so, but this normally causes other issues like one where I teleported a couple blocks when I became able to use it again.

Hope that helps.
